wasm/src/account/private_key.rs

```diff
@@ -44,6 +47,17 @@ impl PrivateKey {
         Self { private_key }
     }
 
+    #[wasm_bindgen]
+    pub fn from_seed(seed: &[u8]) -> Self {
```

Comment:
In #270 , there was a comment about turning this into a u64. That wouldn't be secure. This way provides up to 32 bytes of entropy for the seed (instead of 8).